setup_wordpress.sh

```diff
@@ -1,3 +1,91 @@
 #!/bin/bash
```

Comment:
Could you please checkout latest `master` to your branch?

It will add `set -e` command to the beginnging of this file. So that bash will exit on first error.

Reply:
I am going to figure out how to do that as well as make my script a bit more robust.
Git in general is a bit of a maze for me still so I need to figure out what some of the commands mean and do.
Thank you for checking my code so thorougly.
Kind regards Teerminuz